Sources/Core/AST/Stmt/ConditionalCompilationStmt.swift

```diff
@@ -94,10 +94,63 @@ public struct ConditionalCompilationStmt: Stmt {
 
   }
 
+  /// Abstracts the whole condition structure and its sub-conditions.
+  public indirect enum ConditionTree: Codable {
+
+    case operand(Condition)
+
+    case and(ConditionTree, ConditionTree)
+
+    case or(ConditionTree, ConditionTree)
+
+    public enum ConditionKind: String {
+      case holdOnly
+      case skipMain
+      case skipElse
+    }
+
+    /// Visit the SequenceCondition tail to calculate the right case for executing the right branch.
```

Comment:
```suggestion
    /// Visits the SequenceCondition tail to calculate the right case for executing the right branch.
```
Please document what `conditionCase` mean for this function.